src/Symfony/Component/Cache/Traits/RedisTrait.php

```diff
@@ -187,7 +188,7 @@ public static function createConnection($dsn, array $options = [])
                 }
 
                 try {
-                    @$redis->{$connect}($host, $port, $params['timeout'], (string) $params['persistent_id'], $params['retry_interval'], $params['read_timeout']);
+                    @$redis->{$connect}($host, $port, $params['timeout'], (string) $params['persistent_id'], $params['retry_interval'], $params['read_timeout'], $params['ssl'] ?? null);
```

Comment:
If we trust https://github.com/phpredis/phpredis/issues/1600#issuecomment-776109815
this should be `['stream' => $params['ssl'] ?? []]`
would you be able to confirm?

Reply:
No, Without 'stream'. The array provided directly targets redis_set_context_options. The correct value should be an array of php ssl socket context options.

https://www.php.net/manual/en/context.ssl.php